plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -310,75 +334,34 @@ class StaticHtmlReporter : Reporter {
 
     private fun DIV.issueTable(issueSummary: IssueTable) {
         h2 {
-            id = "issue-summary"
-            with(issueSummary) {
-                +"Issue Summary ($errorCount errors, $warningCount warnings, $hintCount hints to resolve)"
-            }
+            id = issueSummary.id()
+            +issueSummary.title()
         }
 
         p { +"Issues from excluded components are not shown in this summary." }
 
-        h3 { +"Packages" }
-
         table("ort-report-table") {
             thead {
                 tr {
                     th { +"#" }
                     th { +"Package" }
-                    th { +"Analyzer Issues" }
-                    th { +"Scanner Issues" }
+                    th { +"Message" }
                 }
             }
 
             tbody {
                 issueSummary.rows.forEachIndexed { rowIndex, issue ->
-                    issueRow(rowIndex + 1, issue)
+                    issueRow(issueSummary.rowId(rowIndex + 1), rowIndex + 1, issue)
                 }
             }
         }
     }
 
-    private fun TR.listIssues(issues: SortedMap<Identifier, List<ResolvableIssue>>) {
-        td {
-            issues.forEach { (id, issues) ->
-                a("#${id.toCoordinates()}") { +id.toCoordinates() }
-
-                ul {
-                    issues.forEach { issue ->
-                        li {
-                            issueDescription(issue)
-                            p { +issue.resolutionDescription }
-                        }
-
-                        if (!issue.isResolved && issue.howToFix.isNotBlank()) {
-                            details {
-                                unsafe { +"<summary>How to fix</summary>" }
-                                markdown(issue.howToFix)
-                            }
-                        }
-                    }
-                }
-            }
-        }
-    }
-
-    private fun TBODY.issueRow(rowIndex: Int, row: ReportTableModel.IssueRow) {
-        val rowId = "issue-$rowIndex"
-
-        val issues = (row.analyzerIssues + row.scanIssues).flatMap { it.value }
-
-        val worstSeverity = issues.filterNot { it.isResolved }.maxOfOrNull { it.severity } ?: Severity.ERROR
-
-        val areAllResolved = issues.isNotEmpty() && issues.all { it.isResolved }
-
-        val cssClass = if (areAllResolved) {
-            "ort-resolved"
-        } else {
-            when (worstSeverity) {
-                Severity.ERROR -> "ort-error"
-                Severity.WARNING -> "ort-warning"
-                Severity.HINT -> "ort-hint"
-            }
+    private fun TBODY.issueRow(rowId: String, rowIndex: Int, row: ReportTableModel.IssueRow) {
+        val cssClass = when (row.issue.severity) {
+            Severity.ERROR -> "ort-error"
+            Severity.WARNING -> "ort-warning"
+            Severity.HINT -> "ort-hint"
```

Comment:
Should this also become something like `"ort-${severity.name.lowercase()}"`?

Reply:
In the other cases where I've just switched form hard-coding to a bit more magic, I was already not 100% convinced. Here in this case I prefer to keep the hard-coded values, because there need to be aligned with what is in the CSS file. As-is, it is still easy to search for the usages of the CSS classes in the static HTML report. So, I'd prefer leaving this as-is.